README.md

```diff
@@ -195,6 +215,23 @@ Count number of events in each queue:
 jq 'map_values(map(keys[0]))' queue_dump.json
 ```
 
+### Debug console
+
+If the configuration option `console.enabled` is set to `true`, a unix socket named `debug.socket` by default can be found next to the configuration while the node is running. It can be connected to by tools like `socat`:
+
+```sh
+$ socat - unix:/path/to/debug.socket
+```
+
+Entering `help` will show available commands. The interface can also be scripted, it may be helpful to change some of the per-connection settings in this case:
+
+```sh
+echo -e 'set -q true -o bincode\nexample-command' | socat - unix-client:/path/to/debug.socket > output.dump
+```
+
+This calls `example-command` on the console, but disables the command confirmation (`-q true`) and changes the output format to bincode (`-o bincode`) before doing so.
+
+
```

Comment:
It's probably just me being dumb but I wanted to play with it. Connected just fine. Then I went ahead and tried `help`, which gave me a list of allowed commands. I wanted to see how to change a session so I typed:
```
set help

ERR  error: Found argument 'help' which wasn't expected, or isn't valid in this context

USAGE:
    casper-console set [OPTIONS]

For more information try --help
```
See under `USAGE`. It sounded like I should try `casper-console set` but that failed:
```
casper-console set
ERR  error: Found argument 'casper-console' which wasn't expected, or isn't valid in this context

USAGE:
    casper-console <SUBCOMMAND>

For more information try --help
```

What I'm trying to say is that we should try to change the `USAGE` output as it's misleading.

Reply:
Actually, I'm not really sure how I should be using the console once I'm already connected. I can't see a README on it anywhere and the auto-generated descriptions aren't very helpful.

Reply:
You entered three commands, `set -o interactive`, `dump-consensus` and an empty command ``.

`era era 45: TBD` is the "interactive" description of the era dump. It is lacking in detail, because the `Display` implementation is still poor.

The data sent afterwards is what you get when you write `help` or are missing a command.

Reply:
> `era era 45: TBD` is the "interactive" description of the era dump. It is lacking in detail, because the Display implementation is still poor.

Ok. That solves the mystery :)